src/Core/Domain/CmsPage/Exception/CannotEditCmsPageException.php

```diff
@@ -31,4 +31,23 @@
  */
 class CannotEditCmsPageException extends CmsPageException
 {
+    /**
+     * A CMS page that is the same everywhere must be edited in all shop context
+     */
+    const EDIT_IN_ALL_SHOPS = 1;
```

Comment:
Please add the scope of the constant.

Reply:
Ok, I guess you mean `public`. By the way, the context is missing on other exception classes too.  I copied form elsewhere.